Usiqee/Module/AnimatedLaunchScreen/AnimateLaunchScreenVC.swift

```diff
@@ -53,6 +53,7 @@ class AnimateLaunchScreenVC: UIViewController {
     
     private func loadUserIfNeeded() {
         ManagerAuth.shared.synchronise { [weak self] in
+            ManagerAuth.shared.didChangeStatus()
```

Comment:
Maybe put this call directly in AuthManager inside the synchronise function ?

Reply:
I wanted to do it too so it will be transparent to view controllers but actually there are 2 different actions, some times synchronise is called while the status isnt changed (refresh user info fg) so the delegate will be fired for no reason. so i splited them into 2 functions so we can control when to call each of them

Reply:
👍